### ISSUE TITLE
The CRD api-group 'kubean.io' is more make sense maybe 

### ISSUE THREAD

**Issue by yankay** [kubean-io/kubean#57]
The kubebean has two api-group  `kubeancluster.kubean.io/v1alpha1`   and `kubeanclusterops.kubean.io/v1alpha1`
```
root@kay161:/tmp/kubean/artifacts# kubectl api-resources |grep bean
kubeanclusters                                 kubeancluster.kubean.io/v1alpha1       false        KuBeanCluster
kubeanclusterops                               kubeanclusterops.kubean.io/v1alpha1    false        KuBeanClusterOps
```
I think it may combine one like `kubean.io/v1alpha1 `, maybe more make sense 

![企业微信截图_16607146493904](https://user-images.githubusercontent.com/1469319/185042757-143ac413-61fb-4f5a-b049-94f1ead70b4f.png)

**Comment by hangscer8** [comment]
I will optimize that.

**Comment by hangscer8** [comment]
fix by #154
